prepare.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/sh
+
+# Dependencies
+
+## These should be audited!
+mymonero_core_cpp_url='https://github.com/ExodusMovement/mymonero-core-cpp'
+mymonero_core_cpp_hash='ade7891183b3b1a36f695fc6a0fd951c484f0e7d'
+monero_core_custom_url='https://github.com/ExodusMovement/monero-core-custom'
+monero_core_custom_hash='c601fdc3a7aa0c449a3e2c99df230f503fb67e3c'
+
+## Boost, hash should match upstream documented
+boost_url='https://dl.bintray.com/boostorg/release/1.69.0/source/boost_1_69_0.tar.gz'
+boost_sha256='9a2c2819310839ea373f42d69e733c339b4e9a19deab6bfec448281554aa4dbb' # per https://www.boost.org/users/history/version_1_69_0.html
+
+if [ "$(basename "$(pwd)")" != "mymonero-core-js" ]; then
+  echo "Should be ran from the repo dir!"
+  exit -1
```

Comment:
```suggestion
  exit 1
```

Should return 0-255. There are similar `exit` calls below.